.zuul/playbooks/containerd-build/run.yaml

```diff
@@ -0,0 +1,19 @@
+- hosts: all
+  become: yes
+  tasks:
+    - name: Build containerd
+      shell:
+        cmd: |
+          set -xe
+          apt-get update
+          apt-get install -y btrfs-tools libseccomp-dev git pkg-config
+          apt-get install -y golang-1.10-go
+          ln -sf ../lib/go-1.10/bin/go /usr/bin/go
+          go get github.com/containerd/containerd
```

Comment:
Don't need to get containerd source code, OpenLab Zuul have checkout containerd code and apply related PR code on it.

Reply:
done